musit-models/src/main/scala/no/uio/musit/functional/MonadTransformers.scala

```diff
@@ -53,6 +55,25 @@ object MonadTransformers {
 
   }
 
+  object MusitResultT {
+
+    /**
+     * Returns a MusitResultT[Future, A] where the Future is successfully
+     * evaluated with the given MusitResult[A] argument.
+     *
+     * @param res the MusitResult to wrap
+     * @param m an implicitly provided Monad[Future]
+     * @tparam A the type we want to access inside the transformer functions
+     * @return {{{MusitResultT[Future, A]}}}
+     */
+    def successful[A](
```

Comment:
I've considered it. But, when considering all the factors at play, I've come to the conclusion that it's  not necessarily a good idea.

We previously had something similar to what's described in that blog post. But, it caused more confusion than necessary. And it was poorly implemented. Being explicit about what's happening helped the understanding of what's going on.